src/service/print/csv.js

```diff
@@ -0,0 +1,298 @@
+const { getData } = require('./csvImporter')
+const JSON2CSV = require('json2csv').parse
+var cheerio = require('cheerio')
+​
+const buildCSVWithCallback = async (id, callback) => {
+  let error = false
+  let errorMsg = ''
+  let totalMarks = 0
+  getData(id)
+    .then(async data => {
+      if (data.error) {
+        callback(null, data.error, data.errorMsg,null)
+      } else {
+        const subject = data.paperData.subject[0]
+        const grade = data.paperData.gradeLevel[0]
+        const examName = data.paperData.name
```

Comment:
@arun-nd Can we add conditional assignment here just in case these attributes are not defined inside `paperData` or the `subject` and `gradeLevel` arrays?